packages/utils/src/SDKErrors.ts

```diff
@@ -250,27 +249,29 @@ export const ERROR_HASH_TYPE: () => SDKError = () => {
   return new SDKError(ErrorCode.ERROR_HASH_TYPE, 'Hash of wrong type')
 }
 
-export const ERROR_HASH_MALFORMED: (hash?: string, type?: string) => SDKError =
-  (hash?: string, type?: string) => {
-    if (hash && type) {
-      return new SDKError(
-        ErrorCode.ERROR_HASH_MALFORMED,
-        `Provided ${type} hash invalid or malformed \nHash: ${hash}`
-      )
-    }
-    if (hash) {
-      return new SDKError(
-        ErrorCode.ERROR_HASH_MALFORMED,
-        `Provided hash invalid or malformed \nHash: ${hash}`
-      )
-    }
-
+export const ERROR_HASH_MALFORMED: (
+  hash?: string,
+  type?: string
+) => SDKError = (hash?: string, type?: string) => {
```

Comment:
fix your IDE !!!1111!!!

Reply:
ok, so for me the IDE and CLI are giving the same result now. And the result is the one committed. Do you get a different one?

Reply:
yeah looks good to me